doc/conf.py

```diff
@@ -30,6 +31,7 @@
 extensions = [
     'recommonmark',
     "sphinx_rtd_theme",
+	'sphinx_markdown_tables',
```

Comment:
```suggestion
    'sphinx_markdown_tables',
```
Do not use any <tab> in Python files.

Reply:
Ok.